thrust/complex.h

```diff
@@ -614,4 +615,3 @@ template <typename T> __host__ __device__ inline bool operator!=(const complex<T
 
 /*! \} // numerics
  */
-
```

Comment:
Please don't eliminate the newlines at the end of these files -- on some platforms, it causes a warning if the last line is not empty.